docs/user/troubleshootingbootstrap.md

```diff
@@ -0,0 +1,220 @@
+
+* `<human readable id>.log`, which contains the log of the container.
+* `<human readable id>.inspect`, which containts the information about the container like the image, volume mounts, arguments etc.
+
+#### directory: bootstrap/journals
+
+The journals directory contains the logs for *important* systemd units. These units are,
+
+* `release-image.log`, the release-image unit is responsible for pulling the Release Image to the bootstrap host.
+* `crio-configure.log` and `crio.log`, these units are responsible for configuring the CRI-O on the bootstrap host and CRI-O daemon respectively.
+* `kubelet.log`, the kubelet service is responsible for running the kubelet on the bootstrap host. The kubelet on the bootstrap host is responsible for running the static pods for etcd, bootstrap-kube-controlplane and various other operators in bootstrap mode.
+* `approve-csr.log`, the approve-csr unit is responsible for allowing control-plane machines to join OpenShift cluster. This unit performs the job of in-cluster approver while the bootstrapping is in progress.
+
+* `bootkube.log`, the bootkube service is the unit that performs the bootstrapping of OpenShift clusters using all the operators. This service is respnsible for running all the required steps to bootstrap the API and then wait for success.
+
+There might also be other services that are important for some platforms like openstack, that will have logs in this directory.
+
+#### directory: bootstrap/pods
+
+The pods directory contains the information and logs from all the render commands for various operators run by the bootkube unit.
+
+For each container the directory has two files,
+
+* `<human readable id>.log`, which contains the log of the container.
+* `<human readable id>.inspect`, which containts the information about the container like the image, volume mounts, arguments etc.
+
+### directory: resources
+
+The resources direcory contains various Kubernetes objects that are present in the cluster. These resources are pulled using the bootstrap API running on the bootstrap host.
+
+### directory: rendered-assets
+
+The rendered-assets directory contains all the files and directories created by the bootkube unit using various render command for operators. This directory is a snapshot of the `/opt/openshift` directory on the bootstrap-host.
+
+### directory: control-plane
+
+The control-plane directory contains logs for each control-plane host. It contains a sub directory for each control-plane host, usually the IP address of the hosts.
+
+```console
+control-plane
+├── 10.0.128.114
+│   ├── containers
+│   ├── failed-units.txt
+│   ├── journals
+│   └── unit-status
+├── 10.0.142.138
+│   ├── containers
+│   ├── failed-units.txt
+│   ├── journals
+│   └── unit-status
+└── 10.0.148.48
+    ├── containers
+    ├── failed-units.txt
+    ├── journals
+    └── unit-status
+
+12 directories, 3 files
+```
+
+#### directory: control-plane/*/containers
```

Comment:
nit: Markdown (at least GitHub's PR/files renderer) thinks this `*` is the beginning of an italics span.  You can backtick your paths like `control-plane/*/containers` to avoid confusing it.